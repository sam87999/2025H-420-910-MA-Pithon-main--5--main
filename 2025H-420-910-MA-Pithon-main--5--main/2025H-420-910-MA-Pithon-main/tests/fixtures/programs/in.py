# Test de Dominic Lemay
liste = ["a","b","c","d","e"]

if "a" in liste:
    print("a est dans la liste")
else:
    print("a n'est pas dans la liste")