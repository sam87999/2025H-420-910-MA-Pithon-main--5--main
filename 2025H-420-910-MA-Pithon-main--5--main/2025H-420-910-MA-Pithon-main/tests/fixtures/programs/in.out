a est dans la liste
